nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/FTPTransfer.java

```diff
@@ -296,13 +296,19 @@ private FileInfo newFileInfo(final FTPFile file, String path) {
         perms.append(file.hasPermission(FTPFile.WORLD_ACCESS, FTPFile.READ_PERMISSION) ? "r" : "-");
         perms.append(file.hasPermission(FTPFile.WORLD_ACCESS, FTPFile.WRITE_PERMISSION) ? "w" : "-");
         perms.append(file.hasPermission(FTPFile.WORLD_ACCESS, FTPFile.EXECUTE_PERMISSION) ? "x" : "-");
+	
+	long lastModifiedTime		= System.currentTimeMillis();
```

Comment:
The formatting is a bit off. Can you remove the `tab` characters and use spaces? Also, the indentation is not quite right.

Reply:
Can you verify the spacing looks good?